packages/01-host/public/index.html

```diff
@@ -1,10 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <script src="http://localhost:3002/remoteEntry.js"></script>
-    <script src="http://localhost:3003/remoteEntry.js"></script>
+    <script type="systemjs-importmap">
+      {
+        "imports": {
+          "app_one": "http://localhost:3001/remoteEntry.js",
+          "app_two": "http://localhost:3002/remoteEntry.js",
+          "app_three": "http://localhost:3003/remoteEntry.js",
+          "app_one/": "http://localhost:3001/",
+          "app_two/": "http://localhost:3002/",
+          "app_three/": "http://localhost:3003/"
+        }
+      }
+    </script>
   </head>
   <body>
     <div id="root"></div>
+    <script src="https://unpkg.com/systemjs/dist/system.js"></script>
+    <script>
+      System.import('app_one')
```

Comment:
is this one necessary? it seems to be importing the `remoteEntry.js` for app_one, but I didn't think that was necessary for the app that's running the `main.js` script to also run the `remoteEntry` script

I could be wrong though.

Reply:
Good question - probably not. I’ll double check once I get a chance. I added it to mirror how the master branch script tags both the remote entry and the main js file

Reply:
I verified it's unneeded - removed it.